src/bin/dual-iir.rs

```diff
@@ -90,6 +99,8 @@ const APP: () = {
             adcs: stabilizer.adcs,
             dacs: stabilizer.dacs,
             clock: stabilizer.cycle_counter,
+            di1: stabilizer.digital_inputs.1,
```

Comment:
I would recommend something more descriptive for this variable name. `di1` is somewhat opaque. What about `digital_in1` or something?